### PR TITLE
Fix index.html

### PR DESCRIPTION
**Pull request by michielbdejong**
Working on a fix for #6.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,14 +16,27 @@
...
+      src="http://solid.github.io/mashlib/dist/mashlib.js"
     ></script>
-    <script type="text/javascript" src="create.js"></script>
+    <script>
+      document.addEventListener('DOMContentLoaded', function() {
+        // panes.runDataBrowser()
+        // Set up the view for the current subject
+        const uri = decodeURIComponent(window.location.hash.substr(1))
+        if (uri.length === 0) {
+          window.location = '?#' + encodeURIComponent('https://michielbdejong.inrupt.net/meetings/Data%20browser%2016%20Dec%202019/')
```
Comment (michielbdejong) [inline index.html]: This is just for testing, need to think about how we want to deal with this.

File: index.html [hunk 2]
```diff
@@ -21,14 +16,27 @@
     </style>
     <script
       type="text/javascript"
-      src="https://timbl.com/timbl/Automation/Library/Mashup/mashlib-prealpha.js"
+      src="http://solid.github.io/mashlib/dist/mashlib.js"
```
Comment (megoth) [inline index.html]: Would it make sense to set up bundle for this version instead of relying on a published version of mashlib?

Comment (michielbdejong) [reply index.html]: Yes! So then this repo needs a build script.
And then it should be requiring solid-ui, not mashlib, see https://github.com/inrupt/data-browser-docs/blob/master/docs/current.md#technical-model

**Comment by michielbdejong** [comment]
We might want to do the following before merging:
* [ ] https://github.com/solid/meeting-pane/pull/7#discussion_r359833699
* [ ] #8 
* [ ] https://github.com/solid/meeting-pane/pull/7/files#r359827387

**Comment by michielbdejong** [comment]
should add webpack to this repo

**Comment by michielbdejong** [comment]
should add webpack to this repo

File: index.html (hunk 2 above)
Comment (timbl) [reply index.html]: Hmmm   ... one advantage of having a common URI for hosted mashlib.js is the caching of the library by browsers when users use more than one app.

File: index.html [hunk 3]
```diff
@@ -21,14 +16,27 @@
...
+        outliner.GotoSubject(subject, true, undefined, true, undefined)
+      })
+    </script>
   </head>
   <body>
     <h1>The Meetulator</h1>
     <p>
-      A handy gadget for keeping all the parts of a meetingin one place where
+      A handy gadget for keeping all the parts of a meeting in one place where
       you -- and others at the meeting can find them
```
Comment (TallTed) [inline index.html]:
```suggestion
      you -- and others at the meeting -- can find them
```

**Comment by michielbdejong** [comment]
Included in #9

File: index.html (hunk 2 above)
Comment (michielbdejong) [reply index.html]: Right, we should tell our various webpack configs to fetch rdflib.js and solid-ui from cdn, see https://webpack.js.org/configuration/externals/. I created #10 about that.

File: index.html (hunk 1 above)
Comment (michielbdejong) [reply index.html]: Following up in #11

**Pull request closed without merging**